apps/api/README.md

```diff
@@ -39,6 +39,7 @@ Available options:
 | `SHARED_BRIDGE_ADDRESS` | Shared Bridge address | N/A | Yes | |
 | `STATE_MANAGER_ADDRESSES` | CSV list of State manager addresses | N/A | Yes | |
 | `L1_RPC_URLS` | CSV list of RPC URLs. For example, `https://eth.llamarpc.com,https://rpc.flashbots.net/fast` | N/A | Yes | You can check [Chainlist](https://chainlist.org/) for a list of public RPCs |
+| `L2_RPC_URLS_MAP` | JSON from chain id to CSV list of L2 RPC URLs. For example, {"324":"https://mainnet.era.zksync.io,https://zksync.drpc.org"} | N/A | No | You can check [Chainlist](https://chainlist.org/) for a list of public RPCs |
```

Comment:
if we are already parsing a json don't you think having an array of rpcs is better ?

Reply:
makes sense 👍

Reply:
replaced also L1_RPC_URLS to also be a json array since it's easier to read